docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
```

Comment:
if there won't be row templating are we need head body? or just can use q-grid:th, q-grid:td?